libs/providers/flagd-web/src/e2e/step-definitions/flag.ts

```diff
@@ -0,0 +1,352 @@
+      details = await client.getStringDetails(flagKey, defaultValue);
+    },
+  );
+
+  then(/^the default string value should be returned$/, () => {
+    expect(details).toBeDefined();
+    expect(details.value).toEqual(fallback);
+  });
+
+  and(
+    /^the reason should indicate an error and the error code should indicate a missing flag with "(.*)"$/,
+    (errorCode: string) => {
+      expect(details).toBeDefined();
+      expect(details.reason).toEqual(StandardResolutionReasons.ERROR);
+      expect(details.errorCode).toEqual(errorCode);
+    },
+  );
+
+  when(
+    /^a string flag with key "(.*)" is evaluated as an integer, with details and a default value (\d+)$/,
+    async (key: string, defaultValue: string) => {
+      flagKey = key;
+      fallback = Number.parseInt(defaultValue);
+      details = await client.getNumberDetails(flagKey, Number.parseInt(defaultValue));
+    },
+  );
+
+  then(/^the default integer value should be returned$/, () => {
+    expect(details).toBeDefined();
+    expect(details.value).toEqual(fallback);
+  });
+
+  and(
+    /^the reason should indicate an error and the error code should indicate a type mismatch with "(.*)"$/,
+    (errorCode: string) => {
+      expect(details).toBeDefined();
+      expect(details.reason).toEqual(StandardResolutionReasons.ERROR);
+      expect(details.errorCode).toEqual(errorCode);
+    },
+  );
+
+  let ran: Promise<boolean>;
+  when('a PROVIDER_READY handler is added', () => {
+    ran = new Promise<boolean>((resolve) => {
+      client.addHandler(ProviderEvents.Ready, async () => {
+        resolve(true);
+      });
+    });
+  });
+  then('the PROVIDER_READY handler must run', () => {
+    expect(ran).toBeTruthy();
+  });
+
+  when('a PROVIDER_CONFIGURATION_CHANGED handler is added', () => {
+    ran = new Promise<boolean>((resolve) => {
+      client.addHandler(ProviderEvents.ConfigurationChanged, async (details) => {
+        // file writes are not atomic, so we get a few events in quick succession from the testbed
+        // some will not contain changes, this tolerates that; at least 1 should have our change
+
+        // All Flags are changed we do not provide a list of changed flags, that is the nature of the web sdk
+        //if (details?.flagsChanged?.length) {
+        //  flagsChanged = details?.flagsChanged;
+
+        resolve(true);
+        //}
+      });
+    });
+  });
+
+  and(/^a flag with key "(.*)" is modified$/, async () => {
+    // this happens every 1s in the associated container, so wait 3s
+    await new Promise((resolve) => setTimeout(resolve, 3000));
+  });
+
+  then('the PROVIDER_CONFIGURATION_CHANGED handler must run', async () => {
+    expect(await ran).toBeTruthy();
+  });
+
+  and(/^the event details must indicate "(.*)" was altered$/, (flagName) => {
+    // All Flags are changed we do not provide a list of changed flags, that is the nature of the web sdk
+    //expect(flagsChanged).toContain(flagName);
```

Comment:
Can this be reenabled?

Reply:
No, we don't support this feature currently. There is no list of changed flags - hence this and the code above are commented out. This would be an enhancement to the provider, and is not part of this gherkin migration